update/v1alpha1/types_update_status_health.go

```diff
@@ -0,0 +1,129 @@
+package v1alpha1
+
+import metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+// HealthInsight is a piece of actionable information produced by an insight producer about the health
+// of the cluster or an update
+type HealthInsight struct {
+	// startedAt is the time when the condition reported by the insight started
+	// +required
+	// +kubebuilder:validation:Type=string
+	// +kubebuilder:validation:Format=date-time
+	StartedAt metav1.Time `json:"startedAt"`
+
+	// scope is list of objects involved in the insight
+	// +required
+	Scope InsightScope `json:"scope"`
+
+	// impact describes the impact the reported condition has on the cluster or update
+	// +required
+	Impact InsightImpact `json:"impact"`
+
+	// remediation contains information about how to resolve or prevent the reported condition
+	// +required
+	Remediation InsightRemediation `json:"remediation"`
+}
+
+// InsightScope is a list of resources involved in the insight
+type InsightScope struct {
+	// type is either ControlPlane or WorkerPool
+	// +required
+	Type ScopeType `json:"type"`
```

Comment:
Why can't a insight impact both control plane and data plane?
Can we define insight?